kodilla-spring/src/main/java/com/kodilla/spring/calculator/Calculator.java

```diff
@@ -0,0 +1,39 @@
+package com.kodilla.spring.calculator;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public final class Calculator
+{
+    @Autowired
+    private Display display = new Display();
```

Comment:
Dlaczego tworzysz tutaj obiekt? Przypisaniem do tego pola instancji powinna zajmować się adnotacja @Autowired

Reply:
W zasadzie to nie wiem, nie wiedziałem że bez tego zadziała, notacja ta była w przykładzie na początku użyta do wskazania właściwej metody a nie tworzeniem instancji i to mnie chyba zmyliło. Temat jeszcze zbyt świeży. Poprawione